app/routes/abstract-module-route.js

```diff
@@ -1,6 +1,8 @@
-import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 import Ember from 'ember';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
```

Comment:
@billybonks I've tried to keep the imports alphabetical.  I'm not sure I see a reason to put it at the top.  I understand and support the other changes, but moving the Ember import feels superfluous.  If I recall correctly I think I heard that at some point the Ember import goes away, so I suppose you could make a small argument to put it at the top for easy removal later?